examples/Console/TestRedis.cs

```diff
@@ -96,16 +96,17 @@ private static void DoWork(IDatabase db, ActivitySource activitySource)
                 catch (ArgumentOutOfRangeException e)
                 {
                     // Set status upon error
-                    activity.AddTag(SpanAttributeConstants.StatusCodeKey, SpanHelper.GetCachedCanonicalCodeString(Status.Internal.CanonicalCode));
-                    activity.AddTag(SpanAttributeConstants.StatusDescriptionKey, e.ToString());
+                    activity.SetTag(SpanAttributeConstants.StatusCodeKey, SpanHelper.GetCachedCanonicalCodeString(Status.Internal.CanonicalCode));
```

Comment:
Should we consider changing the return type of `SpanHelper.GetCachedCanonicalCodeString` to int?

Reply:
Yes it must be changed. Planning to make it as a separaete PR to avoid this one being much more larger.

Reply:
I think status is going away, but if we're keeping it, I don't think it's an int? Status has always shown up as string ("Ok", "Unknown", etc.) hasn't it?

Reply:
On reading again - yes the `Status` concept of OTel is string. But I am less worried about it as its going away.
I was actually trying to say we'll need to replace `http.status_code` attribute to int (and similar changes wherever we were previously using string due to lack of other type support)